src/main/java/com/cascadebot/cascadebot/commands/moderation/KickCommand.java

```diff
@@ -0,0 +1,34 @@
+package com.cascadebot.cascadebot.commands.moderation;
+
+import com.cascadebot.cascadebot.commandmeta.CommandContext;
+import com.cascadebot.cascadebot.commandmeta.ICommandMain;
+import com.cascadebot.cascadebot.commandmeta.Module;
+import com.cascadebot.cascadebot.permissions.CascadePermission;
+import com.cascadebot.cascadebot.utils.DiscordUtils;
+import net.dv8tion.jda.core.entities.Member;
+
+public class KickCommand implements ICommandMain {
+
+    @Override
+    public void onCommand(Member sender, CommandContext context) {
+        Member TargetMember = DiscordUtils.getMember(context.getArg(0),context.getGuild());
```

Comment:
1. camelCase
2. DiscordUtils.getMember can return null
3. you don't know for sure that the user specified another user

Reply:
Alos you sould be combining all the args for getting the user. we have a util method for that